packages/reactivity/src/collectionHandlers.ts

```diff
@@ -349,7 +349,17 @@ function createInstrumentationGetter(isReadonly: boolean, shallow: boolean) {
       return !isReadonly
     } else if (key === ReactiveFlags.IS_READONLY) {
       return isReadonly
-    } else if (key === ReactiveFlags.RAW) {
+    } else if (
+      key === ReactiveFlags.RAW &&
```

Comment:
the proper fix should same as https://github.com/vuejs/core/commit/6ecbd5ce2a7f59314a8326a1d193874b87f4d8c8#diff-a2a50542bbbe44521ef0f006956cf18fe68602c8fa520aefcb32d3addecbc2cfR104

Reply:
Thank you very much for your help. I will make the necessary changes immediately.